### PR TITLE
Diffusion tasks

### PR DESCRIPTION
**Pull request by IlyasMoutawwakil**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/exporters/openvino/convert.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def export(
         )
 
     if "diffusers" in str(model.__class__) and not is_diffusers_available():
-        raise ImportError("The pip package `diffusers` is required to export stable diffusion models to ONNX.")
+        raise ImportError("The package `diffusers` is required to export diffusion models to ONNX.")
```
Comment (echarlaix) [inline optimum/exporters/openvino/convert.py:142]:
```suggestion
        raise ImportError("The package `diffusers` is required to export diffusion models to OpenVINO.")
```

File: optimum/exporters/openvino/__main__.py [hunk 2]
```diff
@@ -317,9 +317,7 @@ class StoreAttr(object):
                 )
             model.config.pad_token_id = pad_token_id
 
-    if "stable-diffusion" in task:
```
Comment (echarlaix) [inline optimum/exporters/openvino/__main__.py:320]: to ensure compatibility with optimum following changes from https://github.com/huggingface/optimum/pull/1947 we need to update things like  https://github.com/huggingface/optimum-intel/blob/cb2f2eca18da8fcddba3ac6544da7a7a07d57f51/optimum/exporters/openvino/__main__.py#L197-L199 and https://github.com/huggingface/optimum-intel/blob/cb2f2eca18da8fcddba3ac6544da7a7a07d57f51/optimum/commands/export/openvino.py#L230 as well

File: optimum/exporters/openvino/__main__.py [hunk 3]
```diff
@@ -47,11 +47,11 @@
 logger = logging.getLogger(__name__)
 
 
-def infer_task(task, model_name_or_path):
+def infer_task(task, model_name_or_path, **kwargs):
```
Comment (echarlaix) [inline optimum/exporters/openvino/__main__.py:50]:
```suggestion
def infer_task(
    task,
    model_name_or_path: str,
    subfolder: str = "",
    revision: Optional[str] = None,
    cache_dir: str = HUGGINGFACE_HUB_CACHE,
    token: Optional[Union[bool, str]] = None,
) -> str:
```

**Comment by IlyasMoutawwakil** [comment]
Only one issue left which seems unrelated to this PR `test_exporters_cli_int4_with_local_model_and_default_config`.

**Review by echarlaix — APPROVED** [review]
Looks good, thanks a lot @IlyasMoutawwakil

**Pull request merged**